boards/risc-v/qemu-rv/rv-virt/src/qemu_rv_appinit.c

```diff
@@ -35,6 +35,7 @@
 #include <nuttx/virtio/virtio-mmio.h>
 
 #include <sys/mount.h>
+#include "riscv_internal.h"
 
```

Comment:
move blank before `#include "riscv_internal.h"`

Reply:
blank line added before it now,